packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -94,6 +114,29 @@ export interface IScrollablePaneStyles {
   contentContainer: IStyle;
 }
 
+/**
+ * {@docCategory ScrollablePane}
+ */
+export enum StickyContainerBehaviorType {
```

Comment:
Per #6386 , we should not be exporting enums. Can this instead be:

    export type StickyContainerBehaviorType = 'default' | 'StickyOnScroll' | 'StickyAlways';

Reply:
Yes please, no more enums. It's inefficient and really awkward as you need to import/export extra things.